### PR TITLE
seafile-shared: 8.0.1 -> 8.0.3, seafile-client: 8.0.1 -> 8.0.3

### PR DESCRIPTION
**Pull request by eduardosm**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by eduardosm** [comment]
Result of `nixpkgs-review pr 128776` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>seafile-client</li>
    <li>seafile-shared</li>
  </ul>
</details>

File: pkgs/applications/networking/seafile-client/default.nix [hunk 1]
```diff
@@ -4,24 +4,15 @@
...
     repo = "seafile-client";
-    rev = "b4b944921c7efef13a93d693c45c997943899dec";
-    sha256 = "2vV+6ZXjVg81JVLfWeD0UK+RdmpBxBU2Ozx790WFSyw=";
+    rev = "v${version}";
+    sha256 = "lhdKbR19ScNeezICf7vwZaeJikPjwbqrz42bo4lhxJs=";
   };
 
-  patches = [
-    # Fix compilation failure with "error: template with C linkage", fixes #122505
-    (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/seafile-client/default.nix:18]: Please remove the unused fetchpatch from inputs.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 128776` at 47236cb7 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>seafile-client</li>
    <li>seafile-shared</li>
  </ul>
</details>

File: pkgs/applications/networking/seafile-client/default.nix (hunk 1 above)
Comment (eduardosm) [reply pkgs/applications/networking/seafile-client/default.nix:18]: Done

**Pull request merged**